File: FibonacciSequence/FibonacciSequence.py
print("How many Fibonacci Numbers would you like to generate?")
n = int(input())
x = 1
previous_x = 0
for loopCount in range(1, n):
    if (loopCount == 1):
        print("0, 1, ")
    elif (loopCount == n-1):
        next_x = x + previous_x
        previous_x = x
        x = next_x    
        print(f"{next_x}")
    else:
        next_x = x + previous_x
        previous_x = x
        x = next_x    
        print(f"{next_x}, ")

File: FibonacciSequence/Fibonacci_Sequence.rb
puts "Which nth fibonacci number would you like to calculate(starts at 0)"
n = gets.chomp.to_i
previous_x = 0
x = 1
(1...n).each do |loop_count|
    if (loop_count == 1)
        print "0, 1, "
    elsif (loop_count == n-1)
        next_x = x + previous_x
        previous_x = x
        x = next_x    
        print "#{next_x}"
    else
        next_x = x + previous_x
        previous_x = x
        x = next_x    
        print "#{next_x}, "
    end  
end